portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Contact Rokiatou Sissoko | Information Science Student</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <script src="script.js"></script>
+  </head>
+  <body> 
+    <div class="top">
+    <header>
+    <img src="images/mintgreen.jpg">
+    </header>
+
+    <nav>
+        <ul>
+            <!-- <li><a href="index.html">Home</a></li> -->
+            <li><a href="index.html">About Me</a></li>
+            <li><a href="works.html">Works</a></li>            
+            <li><a href="contact.html">Contact Me</a></li>
+        </ul>
+     </nav>
+     </div>
+
+    <div class="main">
+      <div class="aboutflex">
+
+        <div class="abouttext">
+          <h1> Contact Me </h1>
+            <p> I aspire to take on projects that will impact millions of users globally. Feel free to reach out to me at rssissoko600@gmail.com for any opportunities or additional questions. </p>
+            
+            </br>
+            </br>
+
+            <div class="row">
+                <div class="column">
+                    <a href="https://www.linkedin.com/in/rokiatousissoko/"> 
+                        <img src="/images/linked.png" style="width:75%"/>
+                    </a> 
+                </div>
+                <div class="column">
+                    <a href="https://github.com/rssissoko600">
+                        <img src="/images/github.png" style="width:75%"/>
+                    </a>
+                </div>
+                <div class="column">
+                    <a href="https://www.facebook.com/rokiatou.sissoko.9026">
+                        <img src="/images/fb.png" style="width:75%"/>
+                    </a>
+                </div>
+            </div>
+          
+         <!-- <img src="/images/rokiaheadshot.jpeg" /> -->
+        </div>
+
+        <!-- <div class="img">
+          <img src="images/cspic.png">
+        </div> -->
+
+     </div>
+
+    <!-- <div class="prideflex">
+      <div class="pridetext">
+      </div>
+    </div> -->
+
+    <div class="contactform">
+
+    <form id="contactform" method="post" action="#">
+      <div class="contanct-title">
+        <h2>Send A Email To Me</h2>
+        <p>I would love to chat!</p>
+      </div>
+
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="name"/>
+        
+        </br>
+        </br>
+        </br>
+
+        <span class="error_message hidden" id="email_error">
+            Please provide your valid email address.
+        </span>
+
+
+        <label for="email">Email:</label>
+        <input type="text" id="email" name="email"/>
+        
+        </br>
+        </br>
+
+        <label for="email">Message:</label>
+        <textarea name="message" rows="4" cols="50"> </textarea>
+        
+        </br>
+        </br>
+
+        <button type="submit">Submit</button>
```

Comment:
What's supposed to happen when I fill out the contact form and click Submit? (I tried, it basically redisplayed the page, but something weird happened with the stylesheet I guess, the graphic icons were distorted..)

Reply:
Working out implementing a confirmation page via Javascript, as of right now it should do nothing.